psp34_shiden/contracts/shiden34/src/lib.rs

```diff
@@ -63,7 +69,37 @@ pub mod shiden34 {
         #[ink(message)]
         fn max_supply(&self) -> u64;
         #[ink(message)]
+        fn price(&self) -> Balance;
```

Comment:
mint_price? otherwise, it's not clear what kind of price, it could be token price.

Reply:
not sure what do you mean. A token has price only during minting. After that price can be set only at marketplace and similar. There is no price for token in the contract besides minting price

Reply:
In that case, `price_per_mint` feild name can be just `price`. the name xxx_price, price_xxx makes me wonder if there's another type of prices (especially in Custom implementation).